src/transformer/utils/get_user.php

```diff
@@ -24,8 +24,9 @@ function get_user(array $config, \stdClass $user) {
 
     if (array_key_exists('send_mbox', $config) && $config['send_mbox'] == true && $hasvalidemail) {
         return [
-            'name' => $fullname,
-            'mbox' => 'mailto:' . $user->email,
+           // 'name' => $fullname,
+			'mbox_sha1sum' => sha1('mailto:' . $user->email),
+           // 'mbox' => 'mailto:' . $user->email,
```

Comment:
No point leaving commented-out code in (it'll be in version control if it's needed).

```suggestion
			'mbox_sha1sum' => sha1('mailto:' . $user->email),
```

Reply:
OK - made this change